Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -665,9 +669,9 @@ instance add_swap_covariantClass : CovariantClass Cardinal Cardinal (swap (· +
   ⟨fun _ _ _ h => add_le_add' h le_rfl⟩
 #align cardinal.add_swap_covariant_class Cardinal.add_swap_covariantClass
 
-instance : CanonicallyOrderedCommSemiring Cardinal.{u} :=
+instance canonicallyOrderedCommSemiring : CanonicallyOrderedCommSemiring Cardinal.{u} :=
   { Cardinal.commSemiring,
-    Cardinal.partialOrder with
+    Cardinal.linearOrder with
```

Comment:
Why are we using the non-computable linear order here?

Reply:
I've reinstated `partialOrder`. Backported in https://github.com/leanprover-community/mathlib/pull/18781.